doc/rtd/development/style_docs.rst

```diff
@@ -115,12 +107,16 @@ source code.
 Common words
 ------------
 
-There are some common words that should follow specific usage:
+There are some common words that should follow specific usage in text:
 
-- ``cloud-init``: Always hyphenated. Follows sentence case, so only
-  capitalised at the start of a sentence (e.g., ``Cloud-init``).
-- ``metadata``, ``datasource``: One word.
-- ``user data``, ``vendor data``: Two words, not to be combined or hyphenated.
+- **Cloud-init**: Always hyphenated, and follows sentence case, so only
```

Comment:
I would leave this lowercase. The point of the bullet is that we generally don't want the name of the project capitalized.